CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+- Validate `tarantool.Interval` limits with the same ruels as in Tarantool.
```

Comment:
```suggestion
- Validate `tarantool.Interval` limits with the same rules as in Tarantool.
```

Reply:
Thank you, added missing CHANGLELOG.md to codespell linting stage